crates/model/src/order.rs

```diff
@@ -677,6 +677,9 @@ pub struct OrderMetadata {
     pub executed_fee_amount: U256,
     #[serde_as(as = "HexOrDecimalU256")]
     pub executed_surplus_fee: U256,
+    #[serde_as(as = "HexOrDecimalU256")]
+    pub executed_fee: U256,
+    pub executed_fee_token: H160,
```

Comment:
Will `executed_surplus_fee` be removed in the next PR?

Reply:
Yes, we want to remove `executed_surplus_fee`, but we can't do it in this PR because frontend needs time to switch to using `executed_fee`.